test/mongoose_push_web/controllers/api_v3_notification_controller_test.exs

```diff
@@ -105,7 +105,7 @@ defmodule MongoosePushWeb.APIv3NotificationControllerTest do
         Jason.encode!(Map.drop(ControllersHelper.silent_request(), ["data"]))
```

Comment:
So you drop `data`, but get that `alert is missing`?

Reply:
Paradoxically yes, because after this drop, the incoming request - from all the mandatory fields - has only `service` field, and in the flow we developed yesterday I match it against `AlertNotification` to simplify the things. The request incoming is just a freshly parsed map, and the whole struct thing is only a conjecture.
This comment is a good catch, but like I mentioned somewhere above - I don't want to waste so much time on tests that are next in line to be rewritten after this story is closed.